samples/drivers/touch/src/main.c

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2024 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(sample, LOG_LEVEL_INF);
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/input/input.h>
+#include <zephyr/drivers/display.h>
+
+#define CROSS_WIDTH  20
+#define CROSS_HEIGHT 20
+#define BUFFER_SIZE  ((CROSS_WIDTH) * (CROSS_HEIGHT) * 4)
```

Comment:
```suggestion
#define BUFFER_SIZE  (CROSS_WIDTH * CROSS_HEIGHT * 4)
```

the convention is to parenthesize the definition, no need to parenthesize everything

Reply:
Fixed